android/lib/src/main/java/com/ern/api/impl/navigation/MenuUtil.java

```diff
@@ -117,7 +117,7 @@ private static MenuItem addButtonAsMenuItem(@NonNull NavigationBarButton button,
     }
 
     public static boolean canLoadIconFromURI(String icon) {
-        return ElectrodeReactContainer.isReactNativeDeveloperSupport() && URLUtil.isValidUrl(icon) && Patterns.WEB_URL.matcher(icon).matches();
+        return ElectrodeReactContainer.isReactNativeDeveloperSupport() && URLUtil.isValidUrl(icon);
```

Comment:
Just curious: Why is this no longer needed?

Reply:
Okay, saw the second commit message. 👍 Maybe we should add some tests?

Reply:
This was returning false for a valid URL. Also, we are checking invalid URL before this and that is good enough for the icon check. If the URL has some issues, we catch the exception anyway and move forward safely.  Many devs have raised concerns with the icons not showing up while connected to the local package manager, and this was the root cause.